### PR TITLE
Adapt identifier and airline style

### PR DESCRIPTION
**Pull request by saschagrunert**
I would like to propose this design change to have a better look and feel in my modest opinion.

You can see in the lower left side of the screenshot that the following changed:

- identifiers are now orange and provide a better visual feedback
- the StatusLine fits into the background and seems less noisy

What do you think about it?

### CODE REVIEW

**Comment by dsifford** [comment]
Thanks for the contribution @saschagrunert.

I do understand and get that the changes add a bit more contrast to Go syntax, but adding this change would move the implementation [out of spec alignment](https://dsifford.github.io/dracula-spec/#sec-Variables) with the other implementations.

The issue with Vim specifically is that its current method of applying syntax highlighting is far more rudimentary to most other editors, so its difficult and/or impossible to scope syntax regions appropriately (see `len`, `append`, and `make` in your example above).

Neovim is currently looking into possibly adding [tree-sitter syntaxes](https://github.com/neovim/neovim/issues/1767#issuecomment-353728712) and dropping the legacy syntax parser which would completely resolve this issue. Not sure where vim proper stands on this.

In short: If this is a must for you, just add your changes into your vim init scripts. 

Sorry for the bad news. Hope you can understand!

**Comment by saschagrunert** [comment]
Alright, thank you for the detailed explanation. 🙏
